tests/test-utils.ts

```diff
@@ -183,6 +183,29 @@ export async function waitForTransferState(
   expect(actualState).toBe(targetState);
 }
 
+export async function waitForFederatedCatalog(
+  client: EdcConnectorClient,
+  targetNbParticipants: number,
```

Comment:
I believe it'd be better to spell Number out directly.
```suggestion
  targetNumberParticipants: number,
```